src/ambient.d.ts

```diff
@@ -0,0 +1,19 @@
+/**
+ * These declarations tell TypeScript that we allow import of images, e.g.
+ * ```
+		<script lang='ts'>
+			import successkid from 'images/successkid.jpg';
+		</script>
+
+		<img src="{successkid}">
+	 ```
+ */
+declare module "*.jpg" {
+	const value: any;
```

Comment:
I think it's always a `string`

Should we also add `jpeg`, `gif`, `svg`, and `webp` ?

Reply:
Oh, yeah, that makes sense. I copied it from somewhere but I guess here it's indeed always a string. Sure, I can add the types.